CONTRIBUTING.md

```diff
@@ -99,11 +99,10 @@ comply with the our guidelines. At the moment not all packages comply with the r
 
 #### Review
 
-Most of the time your pull request will contain mistakes. It's nothing bad, it just happens.
+It's possible (and common) that a pull request will contain mistakes or reviewers will ask for additional tweaks.
+Reviewers will comment on your pull request and point out which changes are needed before the pull request can be merged.
 
-Reviewers will comment on your pull request and point out which changes are needed before the template can be included.
-
-We recommend having only a single commit for pull request, so if you need to make changes in commits but already have a pull request, use the following commands:
+Most PRs will have a single commit, as seen [above](#starting-a-pull-request), so if you need to make changes to the commit but already have a pull request, use the following commands:
```

Comment:
Thanks, I wanted link that recently an coudn't because of "single commit for pull request". How would you like [one more change](https://github.com/Chocimier/void-packages-org/commit/2e6b32a56f7d134484d37f902f0533bca3eb9f2f) in "Committing your changes" and linking there from here?

Reply:
Kinda picked it up but also extended a lot.

Reply:
"per package" was rarely interpreted as "per subpackage". Might we use "per package (with subpackages)" or "per template" or some longer form to make it clearer?